packages/ipfs-utils/src/http.js

```diff
@@ -292,3 +306,38 @@ HTTP.ndjson = ndjson
 HTTP.streamToAsyncIterator = streamToAsyncIterator
 
 module.exports = HTTP
+
+/**
+ * @param {string | URL | Request} resource
+ * @param {APIOptions} options
+ * @returns {Promise<Response>}
+ */
+module.exports.post = (resource, options) => new HTTP().post(resource, options)
+
+/**
+ * @param {string | URL | Request} resource
+ * @param {APIOptions} options
+ * @returns {Promise<Response>}
+ */
+module.exports.get = (resource, options) => new HTTP().get(resource, options)
+
+/**
+ * @param {string | URL | Request} resource
+ * @param {APIOptions} options
+ * @returns {Promise<Response>}
+ */
+module.exports.put = (resource, options) => new HTTP().put(resource, options)
+
+/**
+ * @param {string | URL | Request} resource
+ * @param {APIOptions} options
+ * @returns {Promise<Response>}
+ */
+module.exports.delete = (resource, options) => new HTTP().delete(resource, options)
+
+/**
+ * @param {string | URL | Request} resource
+ * @param {APIOptions} options
+ * @returns {Promise<Response>}
+ */
+module.exports.options = (resource, options) => new HTTP().options(resource, options)
```

Comment:
```suggestion
/**
 * @param {string | URL | Request} resource
 * @param {APIOptions} options
 * @returns {Promise<Response>}
 */
HTTP.post = (resource, options) => new HTTP().post(resource, options)

/**
 * @param {string | URL | Request} resource
 * @param {APIOptions} options
 * @returns {Promise<Response>}
 */
HTTP.get = (resource, options) => new HTTP().get(resource, options)

/**
 * @param {string | URL | Request} resource
 * @param {APIOptions} options
 * @returns {Promise<Response>}
 */
HTTP.put = (resource, options) => new HTTP().put(resource, options)

/**
 * @param {string | URL | Request} resource
 * @param {APIOptions} options
 * @returns {Promise<Response>}
 */
HTTP.delete = (resource, options) => new HTTP().delete(resource, options)

/**
 * @param {string | URL | Request} resource
 * @param {APIOptions} options
 * @returns {Promise<Response>}
 */
HTTP.options = (resource, options) => new HTTP().options(resource, options)

module.exports = HTTP
```

its easier to do typescript stuff like this ...

Reply:
Less good for tree shaking though, you know, when import/export is a thing.

Reply:
yeah but here bundlers wouldn't tree shake much anyway actually only rollup would potentially treeshake something so.. with esm typescript understands exports better and we can do whatever, with cjs we have some limitations sadly.